<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pesquisa</class>
 <widget class="QWidget" name="Pesquisa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Pesquisa</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/iconmonstr-magnifier-1-240.ico</normaloff>icons/iconmonstr-magnifier-1-240.ico</iconset>
  </property>
  <widget class="QComboBox" name="comboBoxTipoPesquisa">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>39</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Nome</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CPF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RG</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelTipoPesquisa">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tipo de Pesquisa:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPesquisa">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>40</y>
     <width>250</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pesquisar</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPesquisa">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>11</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pesquisa:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewPesquisa">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>121</y>
     <width>841</width>
     <height>511</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="lineEditPesquisa">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="checkBoxDadosPessoais">
      <property name="text">
       <string>Dados Pessoais</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxDadosContato">
      <property name="text">
       <string>Dados Contato</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxDadosMoradia">
      <property name="text">
       <string>DadosMoradia</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelImagem">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>120</y>
     <width>177</width>
     <height>236</height>
    </rect>
   </property>
   <property name="text">
    <string>Carregando imagem................</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
